extensions/guacamole-auth-radius/src/licenses/bundled/README

```diff
@@ -0,0 +1,4 @@
+Apache Guacamole includes a number of subcomponents with separate copyright
+notices and license terms. Your use of these subcomponents is subject to the
```

Comment:
I started a discussion on the general list.  In the meantime I removed the assembly/dist.xml file for the extension, and the entries in pom.xml for the assembly.  Are you thinking the entire src/licenses directory should also be removed from the source tree?